lib/livebook/application.ex

```diff
@@ -167,4 +169,12 @@ defmodule Livebook.Application do
   else
     defp app_specs, do: []
   end
+
+  defp storage_specs() do
+    if Mix.env() == :test do
+      []
+    else
+      [Application.fetch_env!(:livebook, :storage)]
```

Comment:
Soon we will need the storage in tests as well, so you should always start it. :) And remember to start it early on, as other processes may rely on it in the future. I would start it right after the repository!

Reply:
I am a little bit lost on how to set it up properly so that it does not conflict with the `Ets` test suits as it starts under the specified name. Is it okay to just `start_link` it inside `setup_all` and leave it (meaning that if is is already started we simply don't care and otherwise when the test ends it will be shut down on its own)?

Reply:
In the test just assume that it is already running and write directly to it. As long as they write to different namespaces, it should be fine. You can also do this trick:

    test "insert/2", config do
      Ets.insert(config.test, ...)
      ...

In other words, make the test name itself the namespace, and that should guarantee uniqueness!